### PR TITLE
Salt module elasticsearch depends on elasticsearch-py

### PR DESCRIPTION
**Pull request by gthvn1**
This patch installs pip and the python client.

### CODE REVIEW

**Comment by elemoine** [comment]
Shouldn't we rely on `pkgs` in `map.jinja` instead? See https://github.com/tcpcloud/salt-formula-elasticsearch/blob/master/elasticsearch/map.jinja#L5-L6.

**Comment by fpytloun** [comment]
Yes, please don't do mess up package-managed system with pip, it's insecure, fragile and dirty.
There should be python-elasticsearch package available.

**Comment by gthvn1** [comment]
Agree. Done.

File: elasticsearch/client.sls [hunk 1]
```diff
@@ -8,6 +8,10 @@
   - user: root
   - group: root
 
+elasticsearch_python_client:
+  pkg.installed:
+  - name: python-elasticsearch
```
Comment (simonpasquier) [inline elasticsearch/client.sls]: better to rely on map.jinja instead of hardcoding the package name here.

**Review by simonpasquier — CHANGES_REQUESTED** [review]

**Review by simonpasquier — APPROVED** [review]

**Pull request merged**